src/gui/plugins/spawn/Spawn.cc

```diff
@@ -186,7 +186,7 @@ void Spawn::LoadConfig(const tinyxml2::XMLElement *)
 }
 
 
-// TODO(ahcorde): Replace this when this function is on ign-rendering6
+// TODO(ahcorde): Replace this when this function is on ign-rendering7
```

Comment:
Unintentional search-and-replace

Reply:
reverted in https://github.com/ignitionrobotics/ign-gazebo/pull/1183/commits/accf027d3417674dd2f48fec374596e723015443